mmedit/datasets/pipelines/augmentation.py

```diff
@@ -626,30 +627,51 @@ class ColorJitter:
 
     Args:
         keys (list[str]): The images to be resized.
-        to_rgb (bool): Whether to convert channels from BGR to RGB.
-            Default: False.
+        channel_order (str): Order of channel, candidates are 'bgr' and 'rgb'.
+            Default: 'rgb'.
     """
 
-    def __init__(self, keys, to_rgb=False, **kwargs):
+    def __init__(self, keys, channel_order='rgb', **kwargs):
         assert keys, 'Keys should not be empty.'
+        assert 'to_rgb' not in kwargs, (
+            '`to_rgb` is not support in ColorJitter, '
+            "which is replaced by `channel_order` ('rgb' or 'bgr')")
 
         self.keys = keys
-        self.to_rgb = to_rgb
+        self.channel_order = channel_order
         self.transform = transforms.ColorJitter(**kwargs)
 
+    def _color_jitter(self, image, this_seed):
+
+        if self.channel_order.lower() == 'bgr':
+            image = image[..., ::-1]
+
+        image = Image.fromarray(image)
+        torch.manual_seed(this_seed)
+        image = self.transform(image)
+        image = np.asarray(image)
+
+        if self.channel_order.lower() == 'bgr':
+            image = image[..., ::-1]
+
+        return image
+
     def __call__(self, results):
+
+        this_seed = random.randint(0, 2**32)
```

Comment:
Why do we need to again random a seed? They are originally random.

Reply:
To ensure all frames are processed by ColorJitter with the same parameters.